src/components/HomeSearchBox/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState } from 'react'
+import useTranslation from 'next-translate/useTranslation'
+
+const HomeSearchBox = () => {
+  const { t } = useTranslation('home')
+  const [searchItem, setSearch] = useState('')
+
+  const handleClickSearch = (e) => {
```

Comment:
I have two concerns here
1. We normally name function that perform an action like this `handle-element-action`. Thus, this case would be `handleSearchClick`
2. Please don't use `e` for event since when adding type, `event` would be more understandable.

Reference below.
<img width="796" alt="Screen Shot 2564-02-27 at 12 47 07" src="https://user-images.githubusercontent.com/32285706/109376861-e8089b00-78f9-11eb-9e35-89b4c21456e1.png">